system/jlib/jptree.cpp

```diff
@@ -7903,10 +7892,14 @@ jlib_decl IPropertyTree * loadConfiguration(const char * defaultYaml, const char
             appendCurrentDirectory(fullpath, false);
             addNonEmptyPathSepChar(fullpath);
             fullpath.append(optConfig);
-            delta.setown(loadConfiguration(fullpath, componentTag));
+            delta.setown(loadConfiguration(fullpath, componentTag, true));
+            globalConfiguration.setown(loadConfiguration(fullpath, "global", false));
         }
         else
-            delta.setown(loadConfiguration(optConfig, componentTag));
+        {
+            delta.setown(loadConfiguration(optConfig, componentTag, true));
```

Comment:
better for these two lines to be outside the if, using a dynamic filename (fullpath or optConfig).